public/app/features/dashboard/components/LibraryPanelCard/LibraryPanelCard.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { useTheme, Icon, IconButton, stylesFactory, TagList, ConfirmModal } from '@grafana/ui';
+import { css, cx } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export interface LibraryPanelCardProps {
+  id: number;
+  uid: string;
+  model: any;
+  title: string;
+  connectedDashboards: number[];
+  varCount: number;
+  lastEdited?: string;
+  lastAuthor?: string;
+  avatarUrl?: string;
+  onClick?: () => void;
+  onDelete?: () => void;
+}
+
+export const LibraryPanelCard: React.FC<LibraryPanelCardProps> = ({
+  title,
+  connectedDashboards,
+  varCount,
+  lastEdited,
+  lastAuthor,
+  children,
+  onClick,
+  onDelete,
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const [showDeletionModal, setShowDeletionModal] = useState(false);
+
+  const onDeletePanel = () => {
+    onDelete?.();
+    setShowDeletionModal(false);
+  };
+
+  return (
+    <>
+      <div className={cx(styles.panelCard)}>
+        <div className={cx(styles.cardHeader)}>
+          <Icon className={cx(styles.panelIcon)} name="book-open" />
+          <div className={cx(styles.wrapper)}>
+            <div className={cx(styles.panelDetails)}>
+              <span>{title}</span>
+              <div className={cx(styles.panelFigures)} onClick={onClick}>
+                <span>Reusable panel</span>
+                <Icon name="apps" className={cx(styles.detailIcon)} />
+                {connectedDashboards.length}
+                <Icon name="x" className={cx(styles.detailIcon)} />
+                {varCount}
+                <span>
+                  Last edited {lastEdited} by {lastAuthor}
+                </span>
```

Comment:
![image](https://user-images.githubusercontent.com/2376619/106438120-57fc4080-6476-11eb-87b1-5156e87ff8b0.png)

From a user perspective would be nice to see what each icon/counter represents. Maybe in a tooltip?